rdbms/src/test/java/org/spine3/server/storage/jdbc/aggregate/query/Given.java

```diff
@@ -34,7 +34,7 @@
  * @author Andrey Lavrov
  */
 @SuppressWarnings("UtilityClass")
-/* package */ class Given {
+class Given {
 
     @SuppressWarnings("StaticNonFinalField")
     private static Logger loggerMock = null;
```

Comment:
Please check whether we actually use this mock in assertions in the tests. If not, let's kill this field..